test/PoolManager.gas.spec.ts

```diff
@@ -1,566 +1,566 @@
+//         await snapshotGasCost(swapExact0For1(expandTo18Decimals(1).div(10000), wallet.address))
+//         expect((await getSlot0()).tick).to.eq(startingTick - 1)
+//       })
+//
+//       it('second swap in block with no tick movement', async () => {
+//         await swapExact0For1(expandTo18Decimals(1).div(10000), wallet.address)
+//         expect((await getSlot0()).tick).to.eq(startingTick - 1)
+//         await snapshotGasCost(swapExact0For1(2000, wallet.address))
+//         expect((await getSlot0()).tick).to.eq(startingTick - 1)
+//       })
+//
+//       it('second swap in block moves tick, no initialized crossings', async () => {
+//         await swapExact0For1(1000, wallet.address)
+//         expect((await getSlot0()).tick).to.eq(startingTick)
+//         await snapshotGasCost(swapExact0For1(expandTo18Decimals(1).div(10000), wallet.address))
+//         expect((await getSlot0()).tick).to.eq(startingTick - 1)
+//       })
+//
+//       it('first swap in block, large swap, no initialized crossings', async () => {
+//         await snapshotGasCost(swapExact0For1(expandTo18Decimals(10), wallet.address))
+//         expect((await getSlot0()).tick).to.eq(-35787)
+//       })
+//
+//       it('first swap in block, large swap crossing several initialized ticks', async () => {
+//         await modifyPosition(startingTick - 3 * tickSpacing, startingTick - tickSpacing, expandTo18Decimals(1))
+//         await modifyPosition(startingTick - 4 * tickSpacing, startingTick - 2 * tickSpacing, expandTo18Decimals(1))
+//         expect((await getSlot0()).tick).to.eq(startingTick)
+//         await snapshotGasCost(swapExact0For1(expandTo18Decimals(1), wallet.address))
+//         expect((await getSlot0()).tick).to.be.lt(startingTick - 4 * tickSpacing) // we crossed the last tick
+//       })
+//
+//       it('first swap in block, large swap crossing a single initialized tick', async () => {
+//         await modifyPosition(minTick, startingTick - 2 * tickSpacing, expandTo18Decimals(1))
+//         await snapshotGasCost(swapExact0For1(expandTo18Decimals(1), wallet.address))
+//         expect((await getSlot0()).tick).to.be.lt(startingTick - 2 * tickSpacing) // we crossed the last tick
+//       })
+//
+//       it('second swap in block, large swap crossing several initialized ticks', async () => {
+//         await modifyPosition(startingTick - 3 * tickSpacing, startingTick - tickSpacing, expandTo18Decimals(1))
+//         await modifyPosition(startingTick - 4 * tickSpacing, startingTick - 2 * tickSpacing, expandTo18Decimals(1))
+//         await swapExact0For1(expandTo18Decimals(1).div(10000), wallet.address)
+//         await snapshotGasCost(swapExact0For1(expandTo18Decimals(1), wallet.address))
+//         expect((await getSlot0()).tick).to.be.lt(startingTick - 4 * tickSpacing)
+//       })
+//
+//       it('second swap in block, large swap crossing a single initialized tick', async () => {
```

Comment:
for what reason is this file being kept?

Reply:
We havent moved over these tests so I think she wanted to keep them but doesnt want to run them :)

Reply:
ya lets keep for reference